toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/SceneView.kt

```diff
@@ -118,20 +118,20 @@ public fun SceneView(
     onViewpointChangedForBoundingGeometry: ((Viewpoint) -> Unit)? = null,
     graphicsOverlays: GraphicsOverlayCollection = rememberGraphicsOverlayCollection(),
     sceneViewProxy: SceneViewProxy? = null,
-    sceneViewInteractionOptions: SceneViewInteractionOptions = SceneViewInteractionOptions(),
-    viewLabelProperties: ViewLabelProperties = ViewLabelProperties(),
-    selectionProperties: SelectionProperties = SelectionProperties(),
+    sceneViewInteractionOptions: SceneViewInteractionOptions = remember { SceneViewInteractionOptions() },
+    viewLabelProperties: ViewLabelProperties = remember { ViewLabelProperties() },
+    selectionProperties: SelectionProperties = remember { SelectionProperties() },
     isAttributionBarVisible: Boolean = true,
     onAttributionTextChanged: ((String) -> Unit)? = null,
     onAttributionBarLayoutChanged: ((AttributionBarLayoutChangeEvent) -> Unit)? = null,
-    cameraController: CameraController = GlobeCameraController(),
+    cameraController: CameraController = remember { GlobeCameraController() },
     analysisOverlays: AnalysisOverlayCollection = rememberAnalysisOverlayCollection(),
     imageOverlays: ImageOverlayCollection = rememberImageOverlayCollection(),
     atmosphereEffect: AtmosphereEffect = AtmosphereEffect.HorizonOnly,
     timeExtent: TimeExtent? = null,
     onTimeExtentChanged: ((TimeExtent?) -> Unit)? = null,
     spaceEffect: SpaceEffect = SpaceEffect.Stars,
-    sunTime: Instant = Instant.parse("2000-09-22T12:00:00Z"),
+    sunTime: Instant = remember { Instant.parse("2000-09-22T12:00:00Z") },
     sunLighting: LightingMode = LightingMode.NoLight,
     ambientLightColor: Color = Color(220, 220, 220, 255),
```

Comment:
As above, I suggest to define a global static property for it:
```suggestion
    ambientLightColor: Color = SceneViewDefaults.DefaultAmbientLightColor,
```